### ISSUE TITLE
More documentation on dependencies

### ISSUE THREAD

**Issue by panakran** [eduardolundgren/gulp-umd#21]
Would it be possible to provide an (realistic) example of how let's say angular1.x and jquery can be included as dependencies, in the string-based and object-based form?

**Comment by alvarotrigo** [comment]
+1

**Comment by Lazerbeak12345** [comment]
yeah... I'm not following the readme very well. Something even just as simple to import as jquery or lodash would be just great - as you could easily see where you put your require statement or whatever.

Specifically, It's unclear on this:

If I have some javascript code in CommonJS format, for example, and run this on it, does it just work? Does that mean I can use typescript to generate that commonjs, then output the correct uml output? (In particular, "uml" output mode on `tsc` doesn't support fallback to globalizing if no module loader is detected so that's why I'm even looking at this project to begin with.)
